Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -39,7 +39,7 @@ AZ_PUSH_DISABLE_DLL_EXPORT_MEMBER_WARNING
 #include <AzAssetBrowser/ui_AzAssetBrowserWindow.h>
 AZ_POP_DISABLE_DLL_EXPORT_MEMBER_WARNING
 
-AZ_CVAR_EXTERNED(bool, ed_useNewAssetBrowserListView);
+AZ_CVAR_API_EXTERNED(AZTF_API, bool, ed_useNewAssetBrowserListView);
```

Comment:
Externed CVARS are now declared this way. The `API` macro depends on which shared library the cvar is declared in. In this case, `ed_useNewAssetBrowserListView` is declared in AzToolsFramework, which uses `AZTF_API` for its API macro.